ToolBox/TradingEconomicsDataDownloader/TradingEconomicsCalendarDownloader.cs

```diff
@@ -190,5 +192,247 @@ public override Task<string> Get(DateTime startUtc, DateTime endUtc)
             var url = $"/calendar/country/all/{startUtc.ToStringInvariant("yyyy-MM-dd")}/{endUtc.ToStringInvariant("yyyy-MM-dd")}";
             return HttpRequester(url);
         }
+
+        /// <summary>
+        /// Represents the Trading Economics Calendar information:
+        /// The economic calendar covers around 1600 events for more than 150 countries a month.
+        /// https://docs.tradingeconomics.com/#events
+        /// </summary>
+        private class TradingEconomicsCalendarRaw
```

Comment:
Is there a reason for having this new class here instead of a new file?

Reply:
I figured that it was the only place where we're ever going to use it, since we transform the data to a new format shortly afterwards and never deal with raw data aside from here. Maybe it might be a worthy endeavor to put it in a new file now that you mention it for future maintenance and extensibility 👍

Reply:
If the only difference between the `Raw` version and the one used by Lean is the strings to decimals, it seems to me it might not be worth adding this extra class? Maybe this `Raw` version could be a derivate of the other one so we avoid some duplication?